src/main/java/com/microsoft/graph/serviceclient/GraphServiceClient.java

```diff
@@ -50,6 +51,17 @@ public GraphServiceClient(@Nonnull RequestAdapter requestAdapter) {
     public GraphServiceClient(@Nonnull AuthenticationProvider authenticationProvider) {
         this(new BaseGraphRequestAdapter(authenticationProvider, null, "v1.0" , getGraphClientOptions()));
     }
+    /**
+     * Instantiate the GraphServiceClient using an AuthenticationProvider, Cloud and OkHttpClient.
+     * @param client The OkHttpClient for the GraphServiceClient.
+     * @param clouds The Clouds for the GraphServiceClient.
+     * @param authenticationProvider The AuthenticationProvider for this GraphServiceClient.
+     *
+     */
+    @SuppressWarnings("LambdaLast")
+    public GraphServiceClient(@Nonnull OkHttpClient client, @Nonnull Clouds clouds, @Nonnull AuthenticationProvider authenticationProvider) {
```

Comment:
The authentication provider parameter should be first to be consistent with other constructors. The suppression you've added should be enough for the build to succeed.
I'm also not sure why a okHttpClient parameter is present since it's not used?